test/ResourceManager.test.cpp

```diff
@@ -2,10 +2,9 @@
 #include <gtest/gtest.h>
 
 
-TEST(ResourceManager, GetResourceStream) {
+TEST(ResourceManager, RefusesAbsolutePaths) {
```

Comment:
I like having ResourceManager on the left since it groups the ResourceManager tests together. Adding the function name starts to bin the tests into too many small categories. For this function, I like having the function name on the right and the description of what is being tested. Not sure beyond that yet though.